src/olympia/lib/settings_base.py

```diff
@@ -425,8 +425,8 @@ def get_db_config(environ_var, atomic_requests=True):
     'corsheaders.middleware.CorsMiddleware',
     # Enable conditional processing, e.g ETags.
     'django.middleware.http.ConditionalGetMiddleware',
-    'olympia.amo.middleware.CommonMiddleware',
     'olympia.amo.middleware.NoVarySessionMiddleware',
+    'olympia.amo.middleware.CommonMiddleware',
```

Comment:
deliberate move?

Reply:
Yeah, https://docs.djangoproject.com/en/2.2/ref/middleware/#middleware-ordering mentions the right place to put `CommonMiddleware` is:
> After `SessionMiddleware` if you’re using `CSRF_USE_SESSIONS`